package.json

```diff
@@ -10,102 +10,81 @@
   "_glotPressUrl": "https://translate.wordpress.org",
   "_glotPressSlug": "wp-plugins/the-events-calendar/stable",
   "_glotPressFileFormat": "%textdomain%-%wp_locale%.%format%",
-  "_glotPressFormats": [
-    "po",
-    "mo"
-  ],
+  "_glotPressFormats": ["po", "mo"],
   "_glotPressFilter": {
     "translation_sets": false,
     "minimum_percentage": 30,
     "waiting_strings": false
   },
+  "_filePath": {
+    "eslint": [
+      "src/modules/**/*.js",
+      "src/resources/js/**/*.js",
+      "!src/resources/js/**/*.min.js",
+      "!src/resources/js/app/**/*.js",
+      "!src/resources/js/views/**/*.js",
+      "!src/resources/js/events-admin.js",
+      "!src/resources/js/tribe-events-bar.js",
+      "!src/resources/js/tribe-events.js",
+      "src/resources/js/views/**/*.js"
+    ],
+    "stylelint": [
+      "src/resources/postcss/**/*.pcss",
+      "!src/resources/postcss/aggregator-page.pcss",
+      "!src/resources/postcss/events-admin.pcss",
+      "!src/resources/postcss/freemius.pcss",
+      "!src/resources/postcss/tribe-events-full-mobile.pcss",
+      "!src/resources/postcss/tribe-events-full.pcss",
+      "!src/resources/postcss/tribe-events-skeleton.pcss",
+      "!src/resources/postcss/tribe-events-theme-mobile.pcss",
+      "!src/resources/postcss/tribe-events-theme.pcss"
```

Comment:
Out of curiosity - while are we excluding so many files?

Reply:
These are old files, and trying to fix the lint issues might break something. So we exclude the old files, but capture the new files.